examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/FileFeedDriver.java

```diff
@@ -45,30 +46,61 @@ public static void main(String[] args) {
 
     private void doMain(String[] args) {
         CmdLineParser parser = new CmdLineParser(this);
+        FeedSocketAdapterClient[] clients = null;
+        String[] adapterUrls = null;
+        int ingestionCount = 0;
         FeedSocketAdapterClient client = null;
         BufferedReader br = null;
+        Random generator = null;
+        boolean multipleClients = false;
+        long startTime = System.currentTimeMillis();
+        long elapsedTime = 0;
+        long elapsedMinuteTime = 0;
+        double ingestionSpeed = 0;
```

Comment:
It's better to define the variable close to where we use it. The AsterixDB practice is a special case that only works under AsterixDB's context.

Reply:
I put it here since try{} and finally{} share the variables. :-)